code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -811,6 +811,10 @@ GLOBAL_LIST_EMPTY(PDAs)
 
 	if(prob(1))
 		message += "\nSent from my PDA"
+	// SKYRAT EDIT BEGIN - PDA messages show a visible message, to comply with CI policy on calling for backup.
+	user.visible_message(span_notice("[user] presses some buttons on [user.p_their()] [src]."), span_notice("You press some buttons on your [src]."))
+	user.balloon_alert_to_viewers("sent a PDA message")
```

Comment:
how would they know this ICly?

Reply:
Seeing as this is to resolve an issue with combat mechanics, and because you can shut the ringer off on PDAs, I want to make sure there's absolutely no chance you can miss the PDA message going out from someone you're looking at, on the off chance there's a ton of common radio spam or you're on a chat tab that filters out the visible_message. There's no way to dodge hearing the radio sound effect short of shutting off your audio entirely, so there shouldn't be a way to dodge the PDA message notification.

Reply:
Ok then have it say "presses some buttons". In fac,t I'd prefer that happened for more than one thing on a PDA.

Reply:
That's against the style guide for balloon alerts. They are short and accurate.

Reply:
I'm not obsfucating that they sent a PDA message when the crux of the issue is silent PDA messaging. You shouldn't have to know that presses some buttons = pda message, it should just communicate what it's supposed to communicate, the PDA message occured

Reply:
> That's against the style guide for balloon alerts. They are short, accurate, and say exactly what happened

Then don't use a balloon alert. Just use the visible message. Or use a sound, or something. I don't like it being such a blatantly OOC message in a balloon alert. Other people would have no way of knowing ICly that they sent a message.